Eatery Watch App Extension/UI/Eateries/CampusEateriesView.swift

```diff
@@ -0,0 +1,201 @@
+//
+//  CampusEateriesView.swift
+//  Eatery Watch App Extension
+//
+//  Created by William Ma on 1/5/20.
+//  Copyright © 2020 CUAppDev. All rights reserved.
+//
+
+import Combine
+import CoreLocation
+import SwiftUI
+
+private class CampusEateriesData: ObservableObject {
+
+    @Published var campusEateries: [CampusEatery] = []
```

Comment:
i haven't dug into SwiftUI yet but just looked up what this is and this is really cool!